spec/features/artisan_questionnaire_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+feature "Artisan Questionnaire" do
+  let(:email) { "bernard.kanyolo+workdey1@andela.com" }
+  let(:password) { "1234567890" }
+
+  background do
+    Capybara.default_driver = :selenium
+    workdey_data = Seed.new
+    workdey_data.create_all
+  end
+
+  scenario "Log in without taking quiz" do
+    sign_in
+    expect(page).to have_content("Artisan Questionnaire")
+  end
+
+  def sign_in
```

Comment:
There is already a `log_in-with(email, password` helper for signing in. It's in support/helpers